.rubocop_todo.yml

```diff
@@ -0,0 +1,719 @@
+    - 'lib/tasks/validate.rake'
+    - 'spec/requests/admin/page/token_field_spec.rb'
+
+# Offense count: 3
+Lint/IneffectiveAccessModifier:
+  Exclude:
+    - 'lib/slices/available_slices.rb'
+    - 'lib/slices/config.rb'
+
+# Offense count: 6
+Lint/ParenthesesAsGroupedExpression:
+  Exclude:
+    - 'spec/lib/slices/has_attachments_integration_spec.rb'
+    - 'spec/models/admin/as_json_spec.rb'
+
+# Offense count: 3
+Lint/ShadowingOuterLocalVariable:
+  Exclude:
+    - 'app/controllers/admin/entries_controller.rb'
+    - 'spec/apis/pages/show_spec.rb'
+    - 'spec/presenters/article_presenter_spec.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+Lint/StringConversionInInterpolation:
+  Exclude:
+    - 'app/models/page.rb'
+    - 'spec/support/request_helpers.rb'
+
+# Offense count: 1
+Lint/UnderscorePrefixedVariableName:
+  Exclude:
+    - 'lib/slices/has_slices.rb'
+
+# Offense count: 10
+# Cop supports --auto-correct.
+# Configuration parameters: IgnoreEmptyBlocks.
+Lint/UnusedBlockArgument:
+  Exclude:
+    - 'app/models/page.rb'
+    - 'app/models/slice.rb'
+    - 'app/presenters/page_presenter.rb'
+    - 'lib/slices/cms_form_builder.rb'
+    - 'lib/slices/localized_fields.rb'
+    - 'lib/slices/paperclip.rb'
+    - 'spec/models/slice/peers_spec.rb'
+    - 'spec/support/matchers.rb'
+
+# Offense count: 20
+# Cop supports --auto-correct.
+# Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
+Lint/UnusedMethodArgument:
+  Enabled: false
+
+# Offense count: 2
+Lint/UselessAccessModifier:
+  Exclude:
+    - 'lib/slices/available_slices.rb'
+    - 'lib/slices/config.rb'
+
+# Offense count: 79
+Lint/UselessAssignment:
+  Enabled: false
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Performance/Detect:
+  Exclude:
+    - 'app/models/layout.rb'
+
+# Offense count: 7
+# Cop supports --auto-correct.
+Performance/RedundantMerge:
+  Exclude:
+    - 'app/controllers/static_assets_controller.rb'
+    - 'app/models/attachment.rb'
+    - 'app/models/slice.rb'
+    - 'lib/slices.rb'
+    - 'lib/slices/page_as_json.rb'
+
+# Offense count: 6
+# Cop supports --auto-correct.
+Performance/StringReplacement:
+  Exclude:
+    - 'app/controllers/admin/assets_controller.rb'
+    - 'app/helpers/admin/admin_helper.rb'
+    - 'app/helpers/admin/assets_helper.rb'
+    - 'app/helpers/navigation_helper.rb'
+    - 'spec/controllers/admin/assets_controller_spec.rb'
+    - 'spec/helpers/navigation_help_spec.rb'
+
+# Offense count: 4
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, Include.
+# SupportedStyles: action, filter
+# Include: app/controllers/**/*.rb
+Rails/ActionFilter:
+  Exclude:
+    - 'app/controllers/admin/admin_controller.rb'
+    - 'app/controllers/admin/pages_controller.rb'
+    - 'app/controllers/slices_controller.rb'
+
+# Offense count: 2
+# Configuration parameters: Include.
+# Include: app/models/**/*.rb
+Rails/HasAndBelongsToMany:
```

Comment:
I think that because Slices runs on mongodb, we should just disable this one fully, in `.rubocop.yml`. HABTM is ok with mongo because there's no "join model" (like you'd have with ActiveRecord). We have disabled this in other projects.